src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -14,16 +13,19 @@
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
 import wolox.training.exceptions.responses.BookIdMismatchException;
 import wolox.training.exceptions.responses.BookNotFoundException;
 import wolox.training.models.Book;
 import wolox.training.repositories.BookRepository;
 
-@Controller
+@RestController
 @RequestMapping("/api/training/books")
 public class BookController {
 
-    private static final String MSG = "Book not found";
+    private static final String MSGNOTFOUND = "Book not found";
```

Comment:
It's nice to have all the constants in a separate file.
In this case, as there are few constants, it will not generate many lines, therefore it will not make the catch very cumbersome. But if you had more constants it would be good if you took them to a different file, mostly for clarity.